app/views/_includes/summary-cards/trainee-progress.html

```diff
@@ -237,7 +239,7 @@
         visuallyHiddenText: "trainee’s start date"
       }
     ]
-  } if canAmend
+  } if canAmend and record.courseDetails.startDate | isInPast()
```

Comment:
```suggestion
  } if canAmend and record.courseDetails.startDate | isInPast
```

Brackets not needed if not passing any arguments